### PR TITLE
Add 2022q2 Azure report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: 2022q2/azure.adoc
Comment (pauamma) [inline 2022q2/azure.adoc:28]: Noting that sponsoring is usually at the end of the section. See report-sample.adoc.

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/azure.adoc
Comment (lwhsu) [reply 2022q2/azure.adoc:19]: I think I'll leave this for the quarterly report maintainers. (and thanks!)

File: 2022q2/azure.adoc
Comment (lwhsu) [reply 2022q2/azure.adoc]: Yeah, but those items are sponsored by different entities so I want to be more explicit.

**Comment by lwhsu** [comment]
Please feel free to modify more before merging. Thanks!

File: 2022q2/azure.adoc
Comment (pauamma) [reply 2022q2/azure.adoc:19]: > I think I'll leave this for the quarterly report maintainers. (and thanks!)

Changed it as lsalvadore confirmed in IRC it should work.

File: 2022q2/azure.adoc
Comment (pauamma) [reply 2022q2/azure.adoc]: > Yeah, but those items are sponsored by different entities so I want to be more explicit.

Left it and added a summary in the usual location.

**Comment by pauamma** [comment]
> Please feel free to modify more before merging. Thanks!

I only saw this after merging and additional edits. Can you confirm they work for you?

**Comment by lwhsu** [comment]
They are fine. Thanks!

**Pull request merged**
